tests/acceptance/tests/Settings/WishlistGuestFunctionalities.spec.ts

```diff
@@ -0,0 +1,62 @@
+    const product2 = await TestDataService.createBasicProduct();
+    const product1Locators = await StorefrontHome.getListingItemByProductId(product1.id);
+    const product2Locators = await StorefrontHome.getListingItemByProductId(product2.id);
+
+    await test.step('Accept all cookies and reload page', async () => {
+        await TestDataService.setSystemConfig({ 'core.basicInformation.acceptAllCookies': true });
+        await ShopCustomer.goesTo(StorefrontHome.url());
+        await StorefrontHome.consentAcceptAllCookiesButton.click();
+        await StorefrontHome.page.reload();
+    });
+
+    await test.step('Add product1 to the wishlist and verify wishlist count updates to 1', async () => {
+        await ShopCustomer.attemptsTo(AddProductToWishlist(product1));
+        await ShopCustomer.expects(product1Locators.wishlistAddedIcon).toBeVisible();
+        await ShopCustomer.expects(StorefrontHome.wishlistBasket).toHaveText('1');
+    });
+
+    await test.step('Add product1 to the cart from wishlist and verify cart total is same with product price', async () => {
+        await ShopCustomer.attemptsTo(AddProductToCartFromWishlist(product1));
+        const productPrice = await product1Locators.productPrice.innerText();
+        const offCanvasSubtotal = await StorefrontOffCanvasCart.subTotalPrice.innerText();
+        ShopCustomer.expects(offCanvasSubtotal).toBe(productPrice);
+        const offcanvasItem = await StorefrontOffCanvasCart.getLineItemByProductNumber(product1.productNumber);
+        await ShopCustomer.expects(offcanvasItem.wishlistAddedButton).toBeVisible();
+    });
+
+    await test.step('Login as customer and verify product1 is still in wishlist', async () => {
+        await ShopCustomer.attemptsTo(Login());
+        await ShopCustomer.goesTo(StorefrontHome.url());
+        await ShopCustomer.expects(StorefrontHome.wishlistBasket).toHaveText('1');
+        await ShopCustomer.expects(product1Locators.wishlistAddedIcon).toBeVisible();       
+    });
+
+    await test.step('Add product2 to the wishlist and verify', async () => {
+        await ShopCustomer.attemptsTo(AddProductToWishlist(product2));
+        await ShopCustomer.expects(product2Locators.wishlistAddedIcon).toBeVisible();   
+    });
+
+    await test.step('Navigate to the wishlist and verify that the products are in correct order', async () => {
+        await StorefrontHome.wishlistIcon.click();
+        await ShopCustomer.expects(StorefrontHome.wishlistBasket).toHaveText('2');
+        await ShopCustomer.expects(StorefrontWishlist.wishListHeader).toBeVisible();
+        const firstProductInListing = StorefrontWishlist.productListItems.first();
+        const firstProductName = await firstProductInListing.locator('.product-name').innerText();
+        const expectedProductName = await product2Locators.productName.innerText();
+        ShopCustomer.expects(firstProductName).toBe(expectedProductName);
```

Comment:
You could compare product name from test data service with UI of storefront.
```suggestion
        ShopCustomer.expects(firstProductName).toBe(product2.name);
```

Reply:
fixed 👍